web-api/openapi/back_office.yaml

```diff
@@ -4,20 +4,39 @@ info:
+    ## Document
+    By document, we refer to a cohesive text, for example a complete news article.
+    However, we only require a simple representation for our system to work. Consisting just of a unique ID, a text snippet and optional properties.
+    The text snippet is ideally a short, meaningful representation of the larger document, reduced to just one or two paragraphs.
 
-    The API offers an endpoint that allows uploading the documents to the system. After a successful upload, it will be possible to use the document in other endpoints.
+    ### Id
+    The document_id is a unique identifier for a single document.
+    We don't require a specific format, but would suggest using UUID for example.
 
-    # Auth
-    To authenticate with the API, a token will be provided.
+    ## Snippet
+    A snippet is a reduced representation of a larger text, for example if the document would be a news article, then its text would be the news article in a pure textual format.
+    In order for our system to work correctly, it is important that the snippet is just the size of one or two paragraphs, and that the snippet's text clearly summarises the larger text.
+    For example, let's take a news article which talks about the effects of inflation.
+    Our snippet could then be: "Inflation worries as prices keep rising. People having budgetary difficulties as a result. Government pressured to take action.".
+    A bad snippet would for example be just taking the very first paragraph of a document, which could sometimes work, but could also lead to:
+    "December 20th 2020. Article written by Jane Doe. On this bright and sunny day, people might forget about their worries sometimes."
+
+    ## Properties
+    Properties are optional entries for documents, which are usually needed to properly show the document back to the user, when it returns as a personalized document.
+    If for example, you'd wish to integrate a type of carousel view, listing a total of 10 personalized documents in a "for you"-section, then you might choose to display each document as an image and title, with a url to link the user to when pressed.
+    For this, we'd need three document properties: `image`, `link` and `title`.
+    You are free to add as many properties as needed.
+
+    ## Auth
+    The API requires an authenticationToken when used.
+    Please see further down in this README.md file, or refer to the test included in this SDK's codebase, on how to set this token correctly.
 
 tags:
   - name: back office
-    description: Management of the documents in the system.
+    description: Manage documents in a typical create/read/update/delete manner.
```

Comment:
"All you to manage the system, including the document and their properties" or something similar?
We changed the name to back office to avoid relating too strongly with only documents. Also, we might add some reading to the front-office part.